### ISSUE TITLE
21 CFR 11 compliant

### ISSUE THREAD

**Issue by nancymishra2301** [reliatec-gmbh/LibreClinica#215]
Hello Team,

I wanted to check if LibreClinica is 21 CRF 11 compliant out of the box. If not, what are the steps we can take to make it 21 CRF 11 compliant.

Thanks
Nancy

**Comment by toskrip** [comment]
Hi Nancy,

I have only limited experience with studies requiring compliance but I will try to answer this question. The LibreClinica the same like its predecessor OpenClinica provides software features, such as personal user accounts and study roles, audit trails and signing of completed eCRFs.

This simply means that the system can be operated in a 21 CFR Part 11 compliant manner (the software features that are necessary for compliance are there). But it does not mean that just by installing LibreClinica you get the compliance out of the box. Part of regulatory compliance is the necessity of written SOPs (data management, but IT operation as well), system validation (qualifications: IQ, OQ, PQ), user training, etc. and these has to be setup for specific installation by the team that operates the EDC system instance.

One may want to also prove that the EDC system is developed with specific quality measures. This is may be a bit complicated as the development is done by community and community cannot be ISO 9001 certified (or at least I do not know about such possibility). However we do also follow standard procedures during the software development and we keep the development process completely transparent. I am actually maintaining our developer documentation here:

https://libreclinica-docs.readthedocs.io

Furthermore we provide some resources that can aid you while validating your system installation.

First is set of documented test scenarios that can be used to define your OQ procedures. These you can find here (is not final, we will be adding more tests in upcoming releases):

https://libreclinica.org/documentation/tests.html

We also publish the results of those tests for every system release:

https://libreclinica.org/documentation/test_results.html

best

Tomas

**Comment by toskrip** [comment]
If you have more questions (that are not tickets), feel free to start discussion in discussions section (which was activated in December 2020). We would like to keep issues only for software related issues (things that will eventually lead to code changes). This is also part of our development processes (traceability of code changes (pull requests) to original issues that are described in reproducible manner.

Tomas

**Comment by oba-reliatec** [comment]
Hello Nancy,

just to confirm and add to Tomas' answer: it is not just the software but also the "surrounding" (which Tomas concisely sketched [above](https://github.com/reliatec-gmbh/LibreClinica/issues/215#issuecomment-754024504)) that needs to be subject to a quality management (QM) system to serve trials compliant to Good Clinical Practice (ICH-GCP), and 21 CFR Part 11. LibreClinica as well as its parent OpenClinica can be operated in such way.
As LibreClinica community we support such endeavour by providing not only the software but also test specifications (see Tests currently on the [documenation page](https://www.libreclinica.org/documentation), as an important part of software validation.
However, to complete a validation and keeping it up by integration into an existing or newly built QM system is always specific to the entity providing the electronic data capture system, i. e. LibreClinica (usually this is the sponsor in the sense of GCP: the entity initiating a trial and holding the ultimate responsibility for it, not neccessarily the one, financing the trial). It takes some experience (or lots of time, and probably trial and error).

There are several companies providing such service. At https://libreclinica.org/support.html you can find some, which can help you depending on your needs and time/staff resources: from "full service" to just parts or assisting you in building most things yourself, using your existing infrastructure. (Note that we do not publish "sponsored links" there, at present the list is limited to substantial contributors, as we want to move forward LibreClinica).
However, you are not restricted to them, there is no "vendor lock-in" when you use LibreClinica.
I hope you enjoy LibreClinica,
Otmar

**Comment by nancymishra2301** [comment]
Thank you very much Tomas and Otmar for your valuable time and explaining in detail. 

By out of the box I meant that the application can be made complaint after configurations which are made available with the shipped code. I am working with my security team on what's the requirements which we need to fulfill and will surely start a discussion as Tomas suggested if needed.  

I have downloaded the documentation and tests available for LibreClinica from the mentioned sites and will keep them handy for security and compliance teams. In addition to this I have already created SOPs for end users so we should be good.

After looking at the details provided I am in better frame of mind then before that we will be able to satisfy the 21 CFR Part 11 compliance. 

Thanks
Nancy
